docs/hw/hookup.md

```diff
@@ -1,87 +1,21 @@
-# iRobot® Create® 3 Hookup Guide
+# iRobot® Create® 3 Hardware Hookup Overview
+The Create® 3 is based on the Roomba®, a robot vacuum cleaner.
```

Comment:
"The Create® 3 robot is based..." (I should have caught this before)

Reply:
done.